### ISSUE TITLE
How to pass `rubocop -d` to ERB Lint like it asks?

### ISSUE THREAD

**Issue by joshuapinter** [Shopify/erb_lint#418]
I'm seeing the following error when running ERB Lint: 

```bash
Linting 1 files with 10 linters...
An error occurred while Style/SafeNavigation cop was inspecting /.../file.html.erb:1:10.
To see the complete backtrace run rubocop -d.

No errors were found in ERB files
```

1. How do I get `erb_lint` to run `rubocop -d` to find out what's causing the error? 
2. ERB Lint is still passing even though it's running into this error - should it not fail if it encounters an error like this?

Thanks!
